services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -232,7 +232,7 @@ class JobStatus(BaseModel):
 
     job_id: UUID
     state: TaskStates
-    progress: PercentageInt = Field(default=0)
+    progress: PercentageInt = Field(default=parse_obj_as(PercentageInt, 0))
```

Comment:
I know that mypy complains about this but I think it is a this a false positive since ``PercentageInt`` is already an ``int``

Moreover, the solution you adopt is IMO very impractical since you adding code to parse ``0`` which is what you expect pydantic to do.

I think the solution is in tuning the pydantic-plugin section of ``mypy.ini``. I have some thoughts, but IMO the way to go here is to mark it with ``type:ignore``